abstract.js

```diff
@@ -210,6 +210,51 @@ function abstractPersistence (opts) {
     })
   })
 
+  testInstance('get client list after subscriptions', function (t, instance) {
+    var client1 = { id: 'abcde' }
+    var client2 = { id: 'efghi' }
+    var subs = [{
+      topic: 'helloagain',
+      qos: 1
+    }]
+
+    instance.addSubscriptions(client1, subs, function (err) {
+      t.notOk(err, 'no error for client 1')
+      instance.addSubscriptions(client2, subs, function (err) {
+        t.notOk(err, 'no error for client 2')
+        var stream = instance.getClientList(subs[0].topic)
+        stream.pipe(concat({encoding: 'buffer'}, function (out) {
```

Comment:
No specific reason. I could change it to string if you like or else let it tend to default